src/App.tsx

```diff
@@ -1,63 +1,112 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
```

Comment:
For the future, better not to use such short names, you can name it `classnames` here